Runtime/Tracking/Velocity/ConstantVelocityTracker.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace Zinnia.Tracking.Velocity
```

Comment:
what's the use case for this class?

Reply:
My move in place uses velocity trackers as speed input. To debug an issue, i use this class to isolate the cause and find out average velocity tracker is part of the problem (frame rate dependant)

Other than debug purpose, this class is for “velocity tracker” whereas “Constant Force” component is for rigidbody. For example, for simulator/non vr mode, I had to simulate a grab and throw. Without this class, we had to key an animation for the interactor to release the object at certain angle and speed. It is a hard way. With this class, we create a constant tracker parented to playAreaAlias, with UseLocal=true, Velocity=(0,0.5,0.5), the object will be thrown consistently to direction desired and speed desired.

Reply:
ok. this class needs a complimenting test to go along with it

Reply:
done